src/Main.java

```diff
@@ -1,278 +1 @@
-package com.company;
 
-import javax.swing.text.StyledEditorKit;
-import java.util.Scanner;
-import java.io.File;
-import java.io.*;
-import java.util.Queue;
-import java.util.*;
-
-class Cell{
```

Comment:
I'm not certain what the use of the "Cell" class is exactly. What is it used for?